pom.xml

```diff
@@ -64,6 +64,11 @@
             <version>0.4.2</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
## Reviewer note:

Adding this dependency because we were using the dependency from the `EasyVCR` pom.xml, we should explicit this out in the CL itself.

Reply:
Question: What is it used for?

Reply:
It's being used in here: https://github.com/EasyPost/easypost-java/blob/758e497082af0fe211a2919400f5c005c06d0660/src/main/java/com/easypost/utils/Cryptography.java#L3